components/form/builder/src/MultiButton/index.js

```diff
@@ -0,0 +1,115 @@
+import {isValidElement, memo} from 'react'
+import PropTypes from 'prop-types'
+import AtomTag from '@s-ui/react-atom-tag'
+import MoleculeField from '@s-ui/react-molecule-field'
+
+import {field, createComponentMemo} from '../prop-types'
+
+const MultiButton = ({
+  alerts,
+  errors,
+  multiButton,
+  onBlur,
+  onChange,
+  onFocus,
+  renderer,
+  tabIndex
+}) => {
+  const errorMessages = errors[multiButton.id]
+  const alertMessages = alerts[multiButton.id]
+
+  const onClickCallback = (id, checked) => {
+    const targetId = id
+
+    let newValue = []
+
+    if (checked) {
+      newValue = multiButton.value.filter(id => id !== targetId)
+    } else {
+      newValue = [...(multiButton.value || []), targetId]
+    }
```

Comment:
In this way we avoid mutations

```suggestion
    const newValue = checked
      ? multiButton.value.filter(id => id !== targetId)
      : [...(multiButton.value || []), targetId]
```